iceoryx_posh/include/iceoryx_posh/internal/mepoo/shared_pointer.hpp

```diff
@@ -26,6 +27,7 @@ namespace mepoo
 {
 enum class SharedPointerError
 {
+    INVALID_STATE,
```

Comment:
Naming inconsistent with other enum values. Do we have an agreement whether we want all caps in enum classes? It is not consistent throughout the repository.

Should not be changed here due to the diff but we need to improve in this regard.

Reply:
From `CONTRIUTING.md`: "_..If in doubt please try to follow them as well as our unwritten conventions in the existing parts of the code base.."_

So yes, `enum classes` are nearly everywhere all caps.